.github/dependabot.yml

```diff
@@ -589,3 +589,33 @@ updates:
     schedule:
       interval: "weekly"
     open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/file_selector/file_selector_android/android"
```

Comment:
Nit: please keep the package entries in this file in alphabetical order; this should go after `espresso`.